### PR TITLE
Fixes #11021: Cannot generate ps1 technique file (missing 50_techniques directory)

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/11021

### CODE REVIEW

File: tools/ncf_dsc.py [hunk 1]
```diff
@@ -25,10 +25,18 @@
 
 # MAIN FUNCTIONS called by command line parsing
 ###############################################
+dsc_ncf_path = "/var/rudder/configuration-repository/dsc/ncf"
+dsc_techniques_path = os.path.join(dsc_ncf_path,"50_techniques")
+rudder_ncf_path = "/var/rudder/configuration-repository/ncf"
+
+def check_dsc_techniques_path():
```
Comment (ncharles) [inline tools/ncf_dsc.py]: why not doing it in the packaging ?

Comment (VinceMacBuche) [inline tools/ncf_dsc.py]: So it fix it even if it deleted after install

Comment (VinceMacBuche) [inline tools/ncf_dsc.py]: We do likewise on rudder-side (metadata.xml)

Comment (VinceMacBuche) [inline tools/ncf_dsc.py]: To check if conf-repo/techniques/ncf_techniques exists

**Pull request merged**